### PR TITLE
Another set of btn conversions to pf4

### PR DESCRIPTION
**Pull request by sg00dwin**
Copy to clipboard btn converted to tertiary button

Screen Shot 2019-09-25 at 5 35 54 PM
Screen Shot 2019-09-25 at 5 36 47 PM (2)

Screen Shot 2019-09-26 at 12 14 22 PM

### CODE REVIEW

File: frontend/public/components/service-catalog/create-binding.tsx [hunk 1]
```diff
@@ -145,8 +145,19 @@ class CreateBindingForm extends React.Component<CreateBindingProps, CreateBindin
...
+                <ActionGroup className="pf-c-form">
+                  <Button
+                    type="submit"
+                    variant="primary">
+                    Create
+                  </Button>
+                  <Button
+                    id="cancel"
+                    onClick={history.goBack}
+                    variant="secondary">
```
Comment (rhamilto) [inline frontend/public/components/service-catalog/create-binding.tsx]: Add `type="button"` so the form doesn't get submitted.

File: frontend/public/components/utils/copy-to-clipboard.tsx [hunk 2]
```diff
@@ -17,10 +17,10 @@ export const CopyToClipboard: React.FC<CopyToClipboardProps> = React.memo((props
     <pre className="co-pre-wrap co-copy-to-clipboard__text">{visibleValue}</pre>
     <Tooltip content={tooltipContent} trigger="click mouseenter focus" exitDelay={1250}>
       <CTC text={props.value} onCopy={() => setCopied(true)}>
-        <button onMouseEnter={() => setCopied(false)} className="btn btn-default co-copy-to-clipboard__btn fix" type="button">
+        <Button variant="tertiary" onMouseEnter={() => setCopied(false)} className=" co-copy-to-clipboard__btn">
```
Comment (rhamilto) [inline frontend/public/components/utils/copy-to-clipboard.tsx]: Add `type="button"`.

**Comment by spadgett** [comment]
> I'm wondering if we should follow the cues in https://patternfly-react.surge.sh/patternfly-4/components/inputgroup/ and soften the border color on the copy to clipboard buttons? Looks a little jarring to me. @spadgett?

+1

**Comment by sg00dwin** [comment]
@rhamilto @spadgett updated copy to clipboard button visual

File: frontend/public/components/utils/_copy-to-clipboard.scss [hunk 3]
```diff
@@ -3,12 +3,10 @@
   position: relative;
 }
 
-.co-copy-to-clipboard__btn {
-  position: absolute;
+.pf-c-clipboard-copy__group-copy.co-copy-to-clipboard__btn {
```
Comment (rhamilto) [inline frontend/public/components/utils/_copy-to-clipboard.scss]: Do you need the extra specificity **and** the `!important`?

File: frontend/public/components/utils/_copy-to-clipboard.scss [hunk 4]
```diff
@@ -25,3 +23,12 @@
...
   }
 }
+
+.pf-c-button.co-copy-to-clipboard__btn,
+.pf-c-button.co-copy-to-clipboard__btn:hover {
+  --pf-c-button--BorderRadius: 0;
+  --pf-c-button--BorderColor: #ccc;
+  padding: 11px 16px;
+  line-height: 20px;
+  --pf-c-button--hover--BorderWidth: 1px;
```
Comment (rhamilto) [inline frontend/public/components/utils/_copy-to-clipboard.scss]: Nit:  alpha

File: frontend/public/components/utils/_copy-to-clipboard.scss (hunk 3 above)
Comment (sg00dwin) [reply frontend/public/components/utils/_copy-to-clipboard.scss]: removed `!important`

**Pull request merged**